tables/twRO/recvpackets.txt

```diff
@@ -1,1548 +1,1583 @@
-088F 2
-0890 2
-0891 2
-0892 2
-0893 2
-0894 2
-0895 2
-0896 2
-0897 2
-0898 2
-0899 2
-089A 2
-089B 2
-089C 2
-089D 2
-089E 2
-089F 2
-08A0 2
-08A1 2
-08A2 2
-08A3 2
-08A4 2
-08A5 2
-08A6 2
-08A7 2
-08A8 2
-08A9 2
-08AA 2
-08AB 2
-08AC 2
-08AD 2
-08AF 10
-08B0 17
-08B1 -1
-08B2 -1
-08B3 -1
-08B4 2
-08B5 6
-08B6 3
-08B8 10
-08B9 12
-08BA 10
-08BB 8
-08BC 10
-08BD 8
-08BE 14
-08BF 8
-08C0 -1
-08C1 2
-08C2 2
-08C3 10
-08C4 8
-08C5 6
-08C6 4
-08C7 -1
-08C8 34
-08C9 2
-08CA -1
-08CB -1
-08CC 109
-08CD 10
-08CE 2
-08CF 10
-08D0 9
-08D1 7
-08D2 10
-08D3 10
-08D4 8
-08D5 -1
-08D6 6
-08D7 28
-08D8 27
-08D9 30
-08DA 26
-08DB 27
-08DC 26
-08DD 27
-08DE 27
-08DF 50
-08E0 51
-08E1 51
-08E2 27
-08E3 157
-08E4 6
-08FC 30
-08FD 6
-08FE -1
-08FF 24
-0900 -1
-0901 -1
-0902 -1
-0903 -1
-0904 -1
-0905 -1
-0906 -1
-0907 5
-0908 5
-090A 26
-090D -1
-090E 2
-090F -1
-0910 10
-0911 30
-0912 10
-0913 30
-0914 -1
-0915 -1
-0916 26
-0917 2
-0918 2
-0919 2
-091A 2
-091B 2
-091C 2
-091D 2
-091E 2
-091F 2
-0920 2
-0921 2
-0922 2
-0923 2
-0924 2
-0925 2
-0926 2
-0927 2
-0928 2
-0929 2
-092A 2
-092B 2
-092C 2
-092D 2
-092E 2
-092F 2
-0930 2
-0931 2
-0932 2
-0933 2
-0934 2
-0935 2
-0936 2
-0937 2
-0938 2
-0939 2
-093A 2
-093B 2
-093C 2
-093D 2
-093E 2
-093F 2
-0940 2
-0941 2
-0942 2
-0943 2
-0944 2
-0945 2
-0946 2
-0947 2
-0948 2
-0949 2
-094A 2
-094B 2
-094C 2
-094D 2
-094E 2
-094F 2
-0950 2
-0951 2
-0952 2
-0953 2
-0954 2
-0955 2
-0956 2
-0957 2
-0958 2
-0959 2
-095A 2
-095B 2
-095C 2
-095D 2
-095E 2
-095F 2
-0960 2
-0961 2
-0962 2
-0963 2
-0964 2
-0965 2
-0966 2
-0967 2
-0968 2
-0969 2
-096A 2
-096B 4
-096C 6
-096D -1
-096E -1
-096F 7
-0970 31
-0971 6
-0972 -1
-0973 7
-0974 2
-0975 -1
-0976 -1
-0977 14
-0978 6
-0979 50
-097A -1
-097B -1
-097C 4
-097D 288
-097E 12
-097F -1
-0980 7
-0981 -1
-0982 7
-0983 29
-0984 28
-0985 -1
-0986 10
-0987 -1
-0988 6
-0989 2
-098A -1
-098B 2
-098C 4
-098D -1
-098E -1
-098F -1
-0990 41
-0991 -1
-0992 -1
-0993 -1
-0994 -1
-0995 -1
-0996 -1
-0997 -1
-0998 8
-0999 11
-099A 9
-099B 8
-099C 6
-099D -1
-099E 12
-099F -1
-09A0 6
-09A1 2
-09A2 6
-09A3 -1
-09A4 18
-09A5 7
-09A6 12
-09A7 10
-09A8 16
-09A9 10
-09AA 16
-09AB 6
-09AC -1
-09AD 12
-09AE 19
-09AF 4
-09B0 10
-09B1 4
-09B2 10
-09B3 6
-09B4 6
-09B5 2
-09B6 6
-09B7 4
-09B8 6
-09B9 4
-09BA 2
-09BB 6
-09BC 6
-09BD 2
-09BE 2
-09BF 4
-09C1 10
-09C2 -1
-09C3 10
-09C4 10
-09C5 1042
-09C6 -1
-09C7 18
-09C8 -1
-09C9 -1
-09CA -1
-09CB 17
-09CC -1
-09CD 8
-09CE 102
-09CF -1
-09D0 -1
-09D1 14
-09D2 -1
-09D3 -1
-09D4 2
-09D5 -1
-09D6 -1
-09D7 -1
-09D8 2
-09D9 4
-09DA -1
-09DB -1
-09DC -1
-09DD -1
-09DE -1
-09DF 7
-09E0 -1
-09E1 8
-09E2 8
-09E3 8
-09E4 8
-09E5 18
-09E6 24
-09E7 3
-09E8 11
-09E9 2
-09EA 11
-09EB -1
-09EC -1
-09ED 3
-09EE 11
-09EF 11
-09F0 -1
-09F1 11
-09F2 12
-09F3 11
-09F4 12
-09F5 11
-09F6 11
-09F7 77
-09F8 -1
-09F9 143
-09FA -1
-09FB -1
-09FC 6
-09FD -1
-09FE -1
-09FF -1
-0A00 269
-0A01 3
-0A02 4
-0A03 2
-0A04 6
-0A05 63
-0A06 6
-0A07 9
-0A08 26
-0A09 55
-0A0A 57
-0A0B 57
-0A0C 66
-0A0D -1

```

Comment:
`inventory_items_nonstackable 0A0D`
we need to check this package. It is missing from the new version of the file